pytradfri/resource.py

```diff
@@ -14,6 +14,7 @@ class ApiResource:
     def __init__(self, raw):
         """Initialize base object."""
         self.raw = raw
+        self.observe_callback = None
```

Comment:
It's not really clear what this is doing, a global observe_callback for all observations?

Reply:
This is only for a specific device/task/group. Not a global for all the objects. At least that's the way I've understand the code so far. Am I wrong about that?

Reply:
My bad, misread where this was. Wouldn't it make more sense to call `observe_callback` with the result?

Reply:
Yeah, which is done in `proces_result` function down below. However I guessed it was a good practice to initialise a variable with a default state in the class constructor. That's why it's set to `None` here.